packages/roosterjs-content-model-dom/lib/modelApi/editing/applyTableFormat.ts

```diff
@@ -261,6 +261,7 @@ export function setFirstColumnFormatBorders(
 
                 switch (rowIndex) {
                     case 0:
+                        cell.isHeader = !!format.hasHeaderRow;
                         break;
                     case 1:
                         setBorderColor(cell.format, 'borderBottom');
```

Comment:
Not relate to this change, but I realize that it is possible there are only 2 rows, in that case, "1" is the same with "rows.length-1", but then there are different handling. Is that ok?

Reply:
Good catch, they should be flipped in order. The order should be row 0, then last row, then row 1, and then any other row.